src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMigrationResource.kt

```diff
@@ -26,6 +28,32 @@ class PrisonPersonMigrationResource(
   private val migrationService: PrisonPersonMigrationService,
   private val migrationHistoryService: MigrationHistoryService,
 ) {
+  @PreAuthorize("hasRole('ROLE_MIGRATE_PRISONPERSON')")
+  @PostMapping
+  @ResponseStatus(value = HttpStatus.ACCEPTED)
+  @Operation(
+    summary = "Starts a prison person migration. The entity type is determined by the migration filter",
+    description = "Starts an asynchronous migration process. This operation will return immediately and the migration will be performed asynchronously. Requires role <b>MIGRATE_PRISONPERSON</b>",
+    responses = [
+      ApiResponse(
+        responseCode = "202",
+        description = "Migration process started",
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized to access this endpoint",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Incorrect permissions to start migration",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+    ],
+  )
+  suspend fun migratePrisonPerson(
+    @RequestBody @Valid migrationFilter: PrisonPersonMigrationFilter,
+  ) = migrationService.startMigration(migrationFilter)
```

Comment:
Can use a generic start migration endpoint for all prison person entities because the migration type is in the migration filter.